src/libretro.c

```diff
@@ -0,0 +1,843 @@
+#include "libretro.h"
+
+#include <stdbool.h>
+#include <time.h>
+#include <string.h>
+#include <math.h>
+
+#include "shared.h"
+#include "sb_types.h"
+#include "gb.h"
+#include "gba.h"
+#include "nds.h"
+
+#define SKYEMU_LIBRETRO_VERSION "0.1.0"
```

Comment:
Can this be based on the GIT_COMMIT_HASH define that we set in our CMake file?

That way we can correlate it back with the regular builds and have it automatically update.

Reply:
sure, I'll look into that